VirtoCommerce.Platform.Data.Notifications/SMSGatewayOptions.cs

```diff
@@ -0,0 +1,22 @@
+namespace VirtoCommerce.Platform.Data.Notifications
+{
+    public class SmsGatewayOptions
```

Comment:
I think that make sense create the two different Options for each provider type and inject or pass into ctor of concrete provider only  settings that it understand.
Factory for this options you can implement in startup as well

Reply:
Committed changes regarding this. If it is too complex - could add two different classes and option sets.